src/test/java/seedu/exercise/ui/testutil/UiPartExtension.java

```diff
@@ -0,0 +1,29 @@
+package seedu.exercise.ui.testutil;
+
+import java.util.concurrent.TimeoutException;
+
+import org.testfx.api.FxToolkit;
+
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import seedu.exercise.ui.UiPart;
+
+/**
+ * Provides an isolated stage to test an individual {@code UiPart}.
+ */
+public class UiPartExtension extends StageExtension {
+    private static final String[] CSS_FILES = {"view/DarkTheme.css", "view/Extensions.css"};
```

Comment:
May need to change the ``DarkTheme.css`` file

Reply:
fixed